usecases/cluster/delegate_test.go

```diff
@@ -202,7 +202,7 @@ func TestDelegateSort(t *testing.T) {
 	delegate.set("N2", NodeInfo{DiskUsage{Available: GB + 128}, now})
 	delegate.set("N3", NodeInfo{DiskUsage{Available: GB + 512}, now})
 	// one block more
-	delegate.set("N4", NodeInfo{DiskUsage{Available: GB + 4096}, now})
+	delegate.set("N4", NodeInfo{DiskUsage{Available: GB + 1<<25}, now})
```

Comment:
what do you think if  we use direct number instead of bit shifting to be easier to reason about by just looking at the number  ?

Reply:
I believe the expression "1<<25" is commonly understood as representing 32MB, but if you feel it would improve clarity, we can certainly change it

Reply:
i think it would improve clarity, but i will leave the decision to you ;)